### PR TITLE
fix: all warnings in the project

### LINKED ISSUE #233: [Filler Task]: Fix all warnings in the project

### PR DESCRIPTION
**Pull request by evavirseda**
Depends on #236 
Fixes #233 

 
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
conflicts with base 🙏🏼

File: components/Footer.tsx [hunk 1]
```diff
@@ -2,37 +2,35 @@ import Link from 'next/link'
...
+    >
+        <div>
+            <Link href='https://github.com/boxfish-studio/sugar-rush/'>
+                <a
+                    target='_blank'
+                    rel='noopener noreferrer'
+                    className='color-fg-on-emphasis text-underline'
+                    style={{ cursor: 'pointer' }}
+                >
+                    Sugar Rush {version}
```
Comment (cpl121) [inline components/Footer.tsx:18]: I receive this warn when starting the project
`warn  - ./components/Footer.tsx
Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)`

I think it is fixed as follows
`import version from '../version.json'` and `Sugar Rush {version.version}`

**Review by cpl121 — CHANGES_REQUESTED** [review]
Nice job, just 1 comment 💯

**Review by cpl121 — APPROVED** [review]
When making a yarn, I see that there are still some warnings, but I don't know if this task only focuses on linter warnings because they are in the task #229 group.
I approved the issue because I think that these warnings could be done in another task, to be able to close the #229
Nice job Eva 🥇

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**